migrations/migration-1-EN163.sql

```diff
@@ -0,0 +1,150 @@
+
+
+-- here path to images after conversion to iiif format
+create table conversion_data_out_file_path
+(
+    conversion_data_id bigint not null,
+    out_file_path      varchar(255) null
+) engine = MyISAM;
+
+create index FK1gps9jp76y93gdm3dn60jy2xk
+    on conversion_data_out_file_path (conversion_data_id);
+
+
+-- enrich task table, should be created before enrich task context -> saved transcription mapping
+create table enrich_task_context
+(
+    id                        bigint not null
+        primary key,
+    task_state                int null,
+    record_id                 bigint null,
+    has_downloaded_enrichment bit    not null,
+    constraint UK_7qo0hlr7qgc7nqtp311d86f5f
+        unique (record_id)
+) engine = MyISAM;
+
+-- this relation stores information which transcriptions have been downloaded so far
+-- this could be done via normal Transcription relation, but we do like to put in DB only
+-- full set of Transcriptions, and not some portion of it, thus we create new relation as temporary
+-- place for them
+create table enrich_task_context_saved_transcriptions
+(
+    enrich_task_context_id  bigint not null,
+    saved_transcriptions_id bigint not null,
+    constraint UK_4pgqvq3orq1tedwxpxnwd7xnu
+        unique (saved_transcriptions_id)
+) engine = MyISAM;
+
+
+-- here we store exception thrown during task processing
+create table persistable_exception_entity
+(
+    id         bigint auto_increment
+        primary key,
+    message    text null,
+    type       varchar(255) null,
+    context_id bigint null
+) engine = MyISAM;
+
+
+-- here context of transcription task. Has no deps on other relations, can be created anytime
+create table transcribe_task_context
+(
+    id                bigint not null
+        primary key,
+    task_state        int null,
+    record_id         bigint null,
+    has_added_failure bit    not null,
+    has_send_record   bit    not null,
+    has_thrown_error  bit    not null,
+    record_json       longtext null,
+    record_json_raw   longtext null,
+    constraint UK_kiqfan5kgnlppn6393mut42k0
+        unique (record_id)
+) engine = MyISAM;
+
+-- in this table are stored update task contexts
+create table update_task_context
+(
+    id                                 bigint not null
+        primary key,
+    task_state                         int null,
+    record_id                          bigint null,
+    has_fetched_updated_transcriptions bit    not null,
+    has_send_updates                   bit    not null,
+    transcription_id                   bigint null,
+    constraint UK_fhm1ro51vm5dxhwceudmv7gkl
+        unique (record_id)
+) engine = MyISAM;
+
+create index FK3mc0r977wrvom35ayyr2stwp5
+    on update_task_context (transcription_id);
+
+-- ALTERING OLD TABLES
+
+
+-- for easier recovery DB will store transcriptions context to not download it at recovery time
+alter table transcription
+    add transcription_content longtext null;
```

Comment:
this type stay as longtext, because transcription content is stored as JSON string, with transcriptions in two formats, and some metadata. To prevent overflow and hard to debug errors it should be longtext